### PR TITLE
10 parse api response - Revision 2

### LINKED ISSUE #10: parse api response
## Description

- parse response from api into a more readable format
- each comment into an array of comment objs (username, text, display seconds)

## Helpful Documentation

Please list any documentation or resources that may be helpful.

- API call can be found here, [server/src/index.ts](https://github.com/natapokie/ai-fashion-mirror/blob/main/server/src/index.ts)

### PR DESCRIPTION
**Pull request by viiiiiicccky**
## Description

**Fixes #10 **

- original PR: https://github.com/natapokie/ai-fashion-mirror/pull/17 
- Added a function to parse the api response and store in an object ResponseData
- Processed the data with parseResponse() on the server socket manager and sent it to the client for display
- **Extra small change**: handled the "Person not found!" api response case

## Documentation Used

Please list any documentation or resources that were referenced while making these changes.

- N/A

## Issues Faced and Resolutions

Please describe any issues you encountered during development and how you resolved them.

- Issue: Description of the issue
    - Resolution: Steps taken to resolve the issue

## How To Test?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- Run the client and the server
- Parsed response should show on localhost:3000 after photo is taken

## Screenshots (if applicable):
Screenshot 2024-08-13 at 9 33 03 PM
Screenshot 2024-08-13 at 9 42 26 PM

### CODE REVIEW

File: server/src/utils/constants.ts
Comment (eriysd) [inline server/src/utils/constants.ts:1]: so organized I love it

**Review by eriysd — APPROVED** [review]
lgtm :)

**Review by natapokie — APPROVED** [review]
looks great! thanks for making those changes vicky

**Review by vicliu2001 — APPROVED** [review]
nice

**Pull request merged**